README.md

```diff
@@ -3,3 +3,24 @@
    <img src="https://avatars.githubusercontent.com/u/1670887?v=4?s=100" width="100px;" alt=""/>
    <br /><sub><b>Pierre bellec</b></sub>
 </a>
+
+Hello, This is my first article
```

Comment:
consider removing. Also I commented on the wrong document by mistake. See my comment in the other file.